core/auth/src/test/java/software/amazon/awssdk/auth/credentials/ProcessCredentialsProviderTest.java

```diff
@@ -132,10 +134,32 @@ public void processOutputLimitIsEnforced() {
                                   .resolveCredentials();
     }
 
+    @Test
+    public void processOutputLimitDefaultPassesLargeInput() {
+
+        final String LONG_SESSION_TOKEN = "lYzvmByqdS1E69QQVEavDDHabQ2GuYKYABKRA4xLbAXpdnFtV030UH4" +
```

Comment:
nit, we don't normally use `final` for local variables. We have check style rule for it, but not for test code.

Reply:
Done